pages/partials/helpLinkPanel.ejs

```diff
@@ -0,0 +1,11 @@
+<div id="linkNav">
+  <% var url = assessment.help_link %>
+  <% if (!assessment.help_link.includes("https://")){%>
+    <% url = "https://" + assessment.help_link%>
+  <% } %>
+  <a href=<%=url%> target="_blank" rel="noopener norefferer">
```

Comment:
```suggestion
  <a href="<%=url%>" target="_blank" rel="noopener noreferrer">
```

Reply:
Done in [1013e62](https://github.com/PrairieLearn/PrairieLearn/pull/6141/commits/1013e621cfe279ce738772d6180d469d083c5bd9)